lumicks/pylake/detail/confocal.py

```diff
@@ -321,7 +321,7 @@ def _get_plot_data(self, channel, adjustment=ColorAdjustment.nothing(), frame=No
         adjustment : lk.ColorAdjustment
             Color adjustments to apply to the output image if channel is set to "rgb".
         """
-        image = getattr(self, f"{channel}_image")
+        image = self.get_image(channel)  # getattr(self, f"{channel}_image")
```

Comment:
Is this comment a leftover from the previous version?
The same comment is also present in lines 361, 369, and 407 in [test_calibrated_images.py](lumicks/pylake/kymotracker/tests/test_calibrated_images.py).